src/ts/components/seekbar.ts

```diff
@@ -416,24 +422,18 @@ export class SeekBar extends Component<SeekBarConfig> {
     let setupMarkers = () => {
       clearMarkers();
 
-      let hasMarkersInUiConfig = uimanager.getConfig().metadata && uimanager.getConfig().metadata.markers
-        && uimanager.getConfig().metadata.markers.length > 0;
-      let hasMarkersInPlayerConfig = player.getConfig().source && player.getConfig().source.markers
-        && player.getConfig().source.markers.length > 0;
-
-      // Take markers from the UI config. If no markers defined, try to take them from the player's source config.
-      let markers = hasMarkersInUiConfig ? uimanager.getConfig().metadata.markers :
-        hasMarkersInPlayerConfig ? player.getConfig().source.markers : null;
-
-      // Generate timeline markers from the config if we have markers and if we have a duration
-      // The duration check is for buggy platforms where the duration is not available instantly (Chrome on Android 4.3)
-      if (markers && player.getDuration() !== Infinity) {
-        for (let marker of markers) {
-          this.timelineMarkers.push({
-            time: 100 / player.getDuration() * marker.time, // convert time to percentage
-            title: marker.title,
-          });
-        }
+      const duration = player.getDuration();
+
+      if (duration === Infinity) {
```

Comment:
I think live videos have `duration == infinity` too, this might break this functionality

Reply:
I just read in the comment section of the isse that `We don't support markers on live streams right now` ... so i guess this is ok

Reply:
This infinity check was there before, I just rearranged the code. Never seemed to be an issue, but you're right regarding live streams. Since we don't support them yet, I'd say we leave that for another PR.